### DISCUSSION TITLE
GPU command line utility integration

### DISCUSSION THREAD

**Discussion by saurabhya** [aristocratos/btop#298] [Ideas]
Hi, is there a way to integrate a command like nvidia-smi in btop++ which will help to view the memory usage and processes on GPU? So, that we can open btop and then observe processes on both cpu as well as gpu.

If not, I think adding this feature will help some of us.

**Comment by ghost** [comment]
I have an old GPU `Mesa Intel(R) HD Graphics 620` and i wanted to monitor its usage too

I found out that maybe i can monitor my GPU using `intel_gpu_top`
